skiko/src/nativeJsMain/cpp/shaper/Shaper.cc

```diff
@@ -36,12 +39,16 @@ SKIKO_EXPORT KNativePointer org_jetbrains_skia_shaper_Shaper__1nMakeShapeThenWra
 SKIKO_EXPORT KNativePointer org_jetbrains_skia_shaper_Shaper__1nMakeShapeDontWrapOrReorder
   (KNativePointer fontMgrPtr) {
     SkFontMgr* fontMgr = reinterpret_cast<SkFontMgr*>((fontMgrPtr));
-    return reinterpret_cast<KNativePointer>(SkShaper::MakeShapeDontWrapOrReorder(SkUnicode::Make(), sk_ref_sp(fontMgr)).release());
+    // TODO: consider if we need/want to use ICU4X or Libgrapheme (skuincode/include has those implementations too)
+    auto unicode = SkUnicodes::ICU::Make();
+    return reinterpret_cast<KNativePointer>(SkShapers::HB::ShapeDontWrapOrReorder(unicode, sk_ref_sp(fontMgr)).release());
 }
 
 SKIKO_EXPORT KNativePointer org_jetbrains_skia_shaper_Shaper__1nMakeCoreText() {
     #ifdef SK_SHAPER_CORETEXT_AVAILABLE
-        return reinterpret_cast<KNativePointer>(SkShaper::MakeCoreText().release());
+        return nullptr;
+        // TODO: build skia with `skia_use_fonthost_mac=true` to have SkShaper::MakeCoreText
```

Comment:
Isn't it a regression?

Reply:
`skia_use_fonthost_mac ` used to be not needed.  So the compilation error was noticed only after skia was built (>3h). 
 The method was not used anywhere, so I added a TODO in case someone has an issue with it.

Reply:
nvm, I'll rebuild skia. We postponed alpha03 and we have time now.

Reply:
Fixed